tools/serverpod_cli/lib/src/database/create_definition.dart

```diff
@@ -1,15 +1,19 @@
+import 'package:serverpod_cli/analyzer.dart';
 import 'package:serverpod_cli/src/analyzer/entities/definitions.dart';
 import 'package:serverpod_shared/serverpod_shared.dart';
 import 'package:serverpod_service_client/serverpod_service_client.dart';
 
 /// Create the target [DatabaseDefinition] based on the [serializableEntities].
 DatabaseDefinition createDatabaseDefinitionFromEntities(
-    List<SerializableEntityDefinition> serializableEntities) {
+  List<SerializableEntityDefinition> serializableEntities,
+  GeneratorConfig config,
```

Comment:
Suggestion: Pass in module name and installed versions as separate parameters.

Since GeneratorConfig contains so much information for the project it becomes hard to detect what is actually required from it in this place. I much destructing the object as early as possible and passing down what is required to the different places. That creates less places to inspect and understand if we would like to change it in the future.